helm-mapkubeapis.yaml

```diff
@@ -0,0 +1,104 @@
+package:
+  name: helm-mapkubeapis
+  version: 0.6.1
+  epoch: 0
+  description: Helm plugin to map and update deprecated Kubernetes APIs in Helm release manifests
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/helm/helm-mapkubeapis
+      tag: v${{package.version}}
+      expected-commit: afa30453b58395ad8215cfe2c1a1edff50428baa
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/docker/docker@v26.0.0-rc3
+        golang.org/x/oauth2@v0.27.0
+        helm.sh/helm/v3@v3.17.4
+
+  # Dropping helm plugins in /usr/libexec since they are binaries and
+  # configuration intended to be run by the helm command.
+  # See https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html
+  # Installers of this package will have to set this environment var
+  # HELM_PLUGINS=/usr/libexec/helm-plugins
+  - runs: |
+      cat Makefile
+      make build
```

Comment:
can we use go/build pipeline rather? Also, we can configure INSTALL_DIR there too by either changing our `output:` or using symlink and you can add another pipeline to copy the config and yamls

Reply:
@mritunjaysharma394 i tried implementing this approach, but keep on see some issue with `go/build` Pipeline
The `go/build` step completes without errors but doesn't create the expected binary:
```
uses: go/build
  with:
    packages: ./cmd/mapkubeapis
    output: usr/libexec/helm-plugins/helm-mapkubeapis/bin/mapkubeapis
    ldflags: -X main.version=v${{package.version}}
```
which results in below error 
`install: can't stat 'usr/libexec/helm-plugins/helm-mapkubeapis/bin/mapkubeapis': No such file or directory`

the pipeline completes without errors but doesn't create the binary file which results the test and other pipeline to fail  due to which i am switch back to the `make build` approach

Reply:
Thanks @Aditvil-Dev i totally overlooked that part! 🙈 i've now updated the pipeline to correctly symlink it to the expected plugin path under `usr/libexec/helm-plugins/...` and also added a step to install the plugin metadata files (plugin.yaml, Map.yaml). everything seems to be working smoothly now